src/cljs/chronograph_web/pages/organization/views.cljs

```diff
@@ -1,10 +1,95 @@
 (ns chronograph-web.pages.organization.views
   (:require [re-frame.core :as rf]
+            [clojure.spec.alpha :as s]
+            [chronograph-web.components.common :as components]
             [chronograph-web.subscriptions :as subs]
             [chronograph-web.pages.organization.events :as org-events]))
 
+(defn- form-valid? [{:keys [name description]}]
+  (and
+   (s/valid? :tasks/name name)
+   (s/valid? :tasks/description description)))
+
+(defn task-form []
+  (let [{:keys [form-params status]} @(rf/subscribe [::subs/create-task-form])
+        form-invalid? (not (form-valid? form-params))
+        {:keys [name description]} form-params]
+    [:form
```

Comment:
Could add some kind of header here indicating that this is the form to create tasks. This is how the page looked for me; it wasn't clear that this was the task creation form.
![Screenshot 2020-10-01 at 4 58 15 PM](https://user-images.githubusercontent.com/8731801/94803687-4db63100-0407-11eb-9050-5b8516f1d0fe.png)